app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, origin, copy = command.split()
+    if command != "cp" or ".txt" in (origin, copy):
+        print("Wrong command")
+        return
```

Comment:
According to task you should do nothing in that case, so you can use reverse condition and don't use additional return)